docs/reference/modules/cluster/remote-clusters-connect.asciidoc

```diff
@@ -0,0 +1,222 @@
+[[remote-clusters-connect]]
+=== Connect to remote clusters
+To connect a remote cluster (Cluster A) to a local cluster (Cluster B), you
+configure Cluster A as a remote on Cluster B.
```

Comment:
I intentionally duplicated the instructions here so that users can connect a remote cluster without having to link to the CCR page. This way, the instructions apply to both CCS and CCR for connecting a remote cluster.